crates/fluvio-version-manager/fixtures/version/stable/manifest.json

```diff
@@ -0,0 +1,4 @@
+{
+  "channel": "stable",
```

Comment:
why channel is this needed here?

Reply:
This help us understand which channel this package points to.
The `manifest.json` is metadata generated when running `fvm install`, so if one runs:
`fvm install stable`, FVM will fetch stable packages and write metadata to `manifest.json` that looks like this.

Reply:
k